package/upgrade/upgrade_manifests.sh

```diff
@@ -1288,6 +1288,14 @@ apply_whereabouts_manifests()
       kubectl apply -f "$manifest"
   done
 
+  echo "Applying cdi manifests"
+
+  for manifest in /usr/local/share/extra_manifests/cdi/*.yaml; do
```

Comment:
upgrade pod may run several times, the better solution is to check if CRD is existing, only apply when none-existing

Reply:
If the CRD is the same, duplicated applies did not bring up any changes.
So, should we focus on the above discussion (for downgrade part)

Reply:
the performance penalty

like in controller, usally call `reflect.DeepEqual` to compare, before really post a request to apiserver